src/smartactions/SchedulingBase.sol

```diff
@@ -168,6 +156,17 @@ abstract contract SchedulingBase is ERC7579ExecutorBase, ISessionValidationModul
         });
     }
 
+    function getAccountJobDetails(
+        address smartAccount,
+        uint256 jobId
+    )
+        external
+        view
+        returns (ExecutionConfig memory)
+    {
+        return _executionLog[smartAccount][jobId];
+    }
+
     function onUninstall() external {
         uint256 count = _accountJobCount[msg.sender];
         for (uint256 i = 1; i <= count; i++) {
```

Comment:
if we start this one at 0 and iterate after first `SSTORE`, it would save some gas